src/cbor/fallback_decoder.rs

```diff
@@ -103,55 +102,6 @@ impl FallbackDecoder {
         }
     }
 
-    /// A heuristic to find the child binary that we’ll use.
-    pub fn locate_child_binary() -> Result<String, String> {
```

Comment:
I think somethink _like_ that might need to stay here, because we still need to be able to select a correct binary in runtime, when we're not being run by `cargo` (e.g. when using Docker, Nix, or just running the executable directly).

E.g. see this after a `cargo build`:

```
❯ ./target/debug/blockfrost-platform --network preview --node-socket-path ~/.local/share/blockfrost-platform/preview/node.socket --solitary

thread '<unnamed>' panicked at src/cbor/fallback_decoder.rs:119:14:
TESTGEN_HS_PATH environment variable not set. Build script may not have run.: NotPresent
```